site/app/templates/course/CourseMaterials.twig

```diff
@@ -316,6 +350,44 @@
         });
     }
 
+    function handleTimeZones() {
+
+        var url = buildUrl({'component': 'misc', 'page': 'get_server_time'});
+
+        $.ajax({
+            type: "POST",
```

Comment:
Note that, although it doesn't hurt to use `POST` in this case, `POST` is generally considered to be creating/updating a resource.

The following content is from https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9.3

The GET method means retrieve whatever information (in the form of an entity) is identified by the Request-URI. If the Request-URI refers to a data-producing process, it is the produced data which shall be returned as the entity in the response and not the source text of the process, unless that text happens to be the output of the process.

The POST method is used to request that the origin server accept the entity enclosed in the request as a new subordinate of the resource identified by the Request-URI in the Request-Line. POST is designed to allow a uniform method to cover the following functions:

      - Annotation of existing resources;
      - Posting a message to a bulletin board, newsgroup, mailing list,
        or similar group of articles;
      - Providing a block of data, such as the result of submitting a
        form, to a data-handling process;
      - Extending a database through an append operation.

Reply:
Noted